internal/build/lifecycle_execution.go

```diff
@@ -40,6 +40,7 @@ type LifecycleExecution struct {
 	mountPaths   mountPaths
 	opts         LifecycleOptions
 	tmpDir       string
+	// DetectOnly   bool
```

Comment:
Do we need this?

Reply:
I don't think so. There are some conflicts as well. WIll tidy it up and and go through everything again and push